fil_fungible_token/src/token/mod.rs

```diff
@@ -272,71 +318,92 @@ where
             )));
         }
 
-        let _old_state = self.state.clone();
+        let old_state = self.state.clone();
```

Comment:
thoughts on alternative patterns for "reverting" here? 

the state struct itself should be rather light however, holding Cids, not hamts

Reply:
Yes, see above